test/test_scripts.py

```diff
@@ -0,0 +1,41 @@
+import os
+import subprocess
+
+import pandas as pd
+import pytest
+
+@pytest.mark.xfail
+def test_combine_features_mock(tmp_path):
```

Comment:
This is just for my learning...but tmp_path, according to what I've read and what you've written here, is an inbuilt Pytest fixture. Is it just enough to import pytest? Is the tmp_path a reserved keyword in pytest? Or can you just replace this with a different name?

Reply:
Yup, pytest will automatically understand the tmp_path is a reserved keyword. 
To use tmp_path, you just need to import pytest.
You can't replace tmp_path with a different name. But you can customize it like this

```
@pytest.fixture
def my_temp_dir(tmp_path):
    return tmp_path

def test_using_my_temp_dir(my_temp_dir):
    ...
```